src/components/manage/Contents/Contents.jsx

```diff
@@ -1314,6 +1314,7 @@ class Contents extends Component {
                               name={configurationSVG}
                               size="24px"
                               color="#826a6a"
+                              className="configurationSVG"
```

Comment:
We don't do camelCased CSS classnames. Dashes always, and trying to be semantic `configuration-icon`. And making sure that is not leaking somewhere else, so be specific.